src/test/resources/fake-agency-interpolated-stop-times/agency.txt

```diff
@@ -0,0 +1,2 @@
+agency_id,agency_name,agency_url,agency_lang,agency_phone,agency_email,agency_timezone,agency_fare_url,agency_branding_url
```

Comment:
Is there a reason that you have unzipped folders instead of zip files?  Is this so that the files are more easily version controlled?  Also, why is the `fake-agency-interpoloated-stop-times.zip` tracked but `fake-agency.zip` is not?

Reply:
Yes, it's so they can be more easily tracked in version control.  I'll get rid of that other zip file, it shouldn't be there.